tp1/freeway/Assets/Scripts/BoardManager.cs

```diff
@@ -11,7 +11,11 @@ public class BoardManager : MonoBehaviour {
 	// 2 = both
 	// 3 = white
 	// 4 = end
-	public GameObject chicken;										//Player
+
+	public static float assetsSize = 0.36f;
+
+	public GameObject chicken;										//Player1
+	public GameObject chicken2;										//Player2
```

Comment:
It would be great it the comments were near the code that affects. We can move it later.